packages/flutter/lib/src/widgets/navigator.dart

```diff
@@ -2411,6 +2457,8 @@ class Navigator extends StatefulWidget {
   /// the initial route.
   ///
   /// If there is no [Navigator] in scope, returns false.
+  ///
+  /// Does not consider [PopScope].
```

Comment:
Below on lines 2473ff does willPop need to be changed to popDisposition in the docs?

Reply:
Yes, thank you!  I updated this and a few other places.